src/sas/qtgui/Perspectives/Fitting/UI/GPUOptionsUI.ui

```diff
@@ -21,7 +21,7 @@
   </property>
   <property name="windowIcon">
    <iconset>
-    <normaloff>../../../../../../../../../../../Users/UI/res/ball.ico</normaloff>../../../../../../../../../../../Users/UI/res/ball.ico</iconset>
+    <normaloff>../../../../../../../../../../../Users/UI/res/sasview-icon.ico</normaloff>../../../../../../../../../../../Users/UI/res/sasview-icon.ico</iconset>
```

Comment:
Most calls to the icon are `<normaloff>:/res/sasview-icon.ico</normaloff>:/res/sasview-icon.ico</iconset>`. Should this and other relative calls be updated?

Reply:
Maybe, though isn't this file retiring soon anyway?